docs/en/stack/security/get-started-tls.asciidoc

```diff
@@ -0,0 +1,429 @@
+[role="xpack"]
+[[get-started-tls]]
+=== Secure communications in the cluster
+
+{security} enables you to encrypt traffic to, from, and within your cluster. 
+The {stack} uses Transport Layer Security (TLS/SSL) to:
+
+. Ensure only authorized nodes can join an {es} cluster.
+. Secure encrypted communications between:
+** {es} nodes
+** Clients and the {es} cluster
+** User web browsers and {kib}
+** Beats and {ls}
+
+//TBD: This tutorial covers xxx
+
+[float]
+[[get-started-tls-prerequisites]]
+==== Before you begin
+
+. Install and configure {es}, {kib}, {ls}, and {metricbeat} as described in 
+<<get-started-elastic-stack>>.  
+
+. Stop {ls}. The method for starting and stopping {ls} varies depending on whether 
+you are running it from the command line or running it as a service. For example, 
+if you are running {ls} from the command line, you can stop it by entering 
+`Ctrl-C`. See {logstash-ref}/shutdown.html[Shutting down {ls}]. 
+
+. Stop {metricbeat}. For example, enter `Ctrl-C` on the command line where it is 
+running.
+
+. Launch the {kib} web interface by pointing your browser to port 5601. For 
+example, http://127.0.0.1:5601[http://127.0.0.1:5601].
+
+. <<get-started-license,Install a trial license>>.
+
+. <<get-started-enable-security,Enable security in {es}>>.
+
+. <<get-started-built-in-users,Create passwords for built-in users>>.
++
+--
+NOTE: This section of the tutorial does not use the built-in `kibana`, 
+`logstash_system`, and `beats_system` users. To learn more about those users, 
+see <<built-in-users>>.
+
+--
+
+. <<get-started-kibana-user,Add the built-in user to {kib}>>.
+
+. To verify that security is enabled, run the cat health API. For example:
++
+--
+[source,sh]
+----------------------------------
+curl -s -u elastic:<password> -X GET "http://localhost:9200/_cat/health?v"
+----------------------------------
+
+Supply a valid user ID (such as `elastic`) and password, such as the one you 
+created in <<get-started-users>>. 
+
+--
+
+[role="xpack"]
+[[get-started-certificates]]
+=== Generate certificates
+
+In a secured cluster, {es} nodes, {kib} instances, {ls} hosts, and Beats use 
+certificates to identify themselves when accepting incoming connections and 
+connecting to other services. 
+
+The cluster must validate the authenticity of these certificates. The 
+recommended approach is to trust a specific certificate authority (CA). Thus 
+when nodes are added to your cluster they just need to use a certificate signed 
+by the same CA and the node is automatically allowed to join the cluster. 
+
+. Pick a verification mode for the certificates. By default, {es} verifies that 
```

Comment:
I'd honestly be in favor of dropping this. Advising users on how to disable hostname verification before they even attempt TLS sets the wrong precedent